### ISSUE TITLE
H4 timeframe

### ISSUE THREAD

**Issue by Raphael0010** [Reiryoku-Technologies/Mida#80]
Hi, there is a way to use H4 timeframe with the MidaTradingSystem ?

Because I have this issue when i run the MidaStradingSystem with H4 period (who is typed)

```
/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/@reiryoku+mida-binance@2.1.0_@reiryoku+mida@2022.5.0/node_modules/@reiryoku/mida-binance/src/core/platforms/binance-spot/BinanceSpotAccount.ts:719
            throw new Error("Unknown timeframe");
                  ^
Error: Unknown timeframe
    at normalizeTimeframe (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/@reiryoku+mida-binance@2.1.0_@reiryoku+mida@2022.5.0/node_modules/@reiryoku/mida-binance/src/core/platforms/binance-spot/BinanceSpotAccount.ts:719:19)
    at BinanceSpotAccount._BinanceSpotAccount_onPeriodUpdate (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/@reiryoku+mida-binance@2.1.0_@reiryoku+mida@2022.5.0/node_modules/@reiryoku/mida-binance/src/core/platforms/binance-spot/BinanceSpotAccount.ts:558:35)
    at /Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/@reiryoku+mida-binance@2.1.0_@reiryoku+mida@2022.5.0/node_modules/@reiryoku/mida-binance/src/core/platforms/binance-spot/BinanceSpotAccount.ts:495:145
    at ReconnectingWebSocket.w.onmessage (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/binance-api-node@0.11.47/node_modules/binance-api-node/dist/websocket.js:219:7)
    at WebSocket.ReconnectingWebSocket._handleMessage (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/reconnecting-websocket@4.4.0/node_modules/reconnecting-websocket/dist/reconnecting-websocket-cjs.js:174:23)
    at WebSocket.onMessage (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/ws@7.5.9/node_modules/ws/lib/event-target.js:132:16)
    at WebSocket.emit (node:events:513:28)
    at WebSocket.emit (node:domain:489:12)
    at Receiver.receiverOnMessage (/Users/quiet/Documents/Perso/ichigram/node_modules/.pnpm/ws@7.5.9/node_modules/ws/lib/websocket.js:1068:20)
    at Receiver.emit (node:events:513:28)
```

The mida system  :  

```ts
import { MidaTradingSystem, MidaTimeframe, } from "@reiryoku/mida";

export class SuperTradingSystem extends MidaTradingSystem {
    watched () {
        return {
            "BTCUSD": {
                watchTicks: true,
                watchPeriods: true,
                timeframes: [ MidaTimeframe.H1, MidaTimeframe.H4, ],
            },
        };
    }
    
    async configure () {
        // Called once per instance before the first startup
        // can be used as async constructor
    }
    
    async onTick (tick) {
        const { bid, ask, date, } = tick;
    }
    
    async onPeriodClose (period) {
        const [ open, high, low, close, ] = period.ohlc;
        
    }
}
``` 

Edit  :  

Found the issue : https://github.com/Reiryoku-Technologies/Mida-Binance/blob/6609378af251cc6f6ec084780bd6ec0f701c8487/src/core/platforms/binance-spot/BinanceSpotAccount.ts#L683

There is no "H4" time frame.

Second edit : 

https://github.com/Reiryoku-Technologies/Mida-Binance/pull/5

**Comment by Vasile-Peste** [comment]
Hello, good catch! Thank you also for creating the PR. I have left a small change request.
Thank you!

**Comment by Vasile-Peste** [comment]
Merged and released with [Mida Binance 2.1.1](https://github.com/Reiryoku-Technologies/Mida-Binance/releases/tag/v2.1.1). Can you update the dependency and let me know that it's working now? Thank you!

**Comment by Raphael0010** [comment]
Thanks for your reactivity @Vasile-Peste it's work !
